templates/UmbracoExtension/Client/package.json

```
@@ -9,13 +9,12 @@
     "generate-client": "node scripts/generate-openapi.js https://localhost:44339/umbraco/swagger/umbracoextension/swagger.json"
   },
   "devDependencies": {
-    "@hey-api/client-fetch": "^0.10.0",
-    "@hey-api/openapi-ts": "^0.66.7",
-    "@umbraco-cms/backoffice": "^UMBRACO_VERSION_FROM_TEMPLATE",
+    "@hey-api/openapi-ts": "^0.80.1",
+    "@umbraco-cms/backoffice": "^16.1.0",
```

This needs to be taken from a variable:

```suggestion
    "@umbraco-cms/backoffice": "^UMBRACO_VERSION_FROM_TEMPLATE",
```